### PR TITLE
complete third task

### PR DESCRIPTION
**Pull request by barsuchellos**
[DEMO LINK](https://barsuchellos.github.io/layout_search-bar-airbnb/)
[TEST REPORT LINK](https://barsuchellos.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,34 @@
...
+      action="url"
+      data-qa="big"
+      method="post"
+      class="container"
+    >
+      <label
+        class="big-search search"
+        data-qa="big"
+        >
+        <img src="../src/images/Search.svg" alt="big-search-logo">
```
Comment (OleksiiBlezniuk) [inline src/index.html:24]: There is no need to use '-' in alt attribute, also it is unnecessary for user read if it's big search logo or small, especially for users with disability, 'Search logo' is enough

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,34 @@
...
+        <input
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          >
```
Comment (OleksiiBlezniuk) [inline src/index.html:29]:
```suggestion
        <input
          type="text"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
```

File: src/index.html [hunk 3]
```diff
@@ -11,6 +11,34 @@
...
+      </label>
+
+      <label
+        class="small-search search"
+        data-qa="small"
+        >
```
Comment (OleksiiBlezniuk) [inline src/index.html:35]:
```suggestion
      <label
        class="small-search search"
        data-qa="small"
      >
```

File: src/index.html [hunk 4]
```diff
@@ -11,6 +11,34 @@
...
+        <img src="../src/images/Search.svg" alt="small-search-logo">
+        <input
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;">
```
Comment (OleksiiBlezniuk) [inline src/index.html:39]:
```suggestion
        <input
          type="text"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```

Comment (OleksiiBlezniuk) [reply src/index.html:39]: Closing bracket should have same indentation as open

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  width: 11px;
+  height: 11px;
+  vertical-align: middle;
+  padding: 14px 9px 15px 12px;
+}
+
+input {
+  font-family: "Avenir" , sans-serif;
+  border: none;
+
```
Comment (OleksiiBlezniuk) [inline src/style.css:44]: Do not use tags for styles, use classes instead

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+.small-search img {
+  width: 11px;
+  height: 11px;
+  vertical-align: middle;
+  padding: 14px 9px 15px 12px;
+}
+
+input {
+  font-family: "Avenir" , sans-serif;
```
Comment (OleksiiBlezniuk) [inline src/style.css:42]: Set font styles in html tag, other elements will inherit it

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+  font-size: 14px;
+  padding: 0 0 0 1px;
+}
+
+input::placeholder {
+  color: #3e4e61;
+}
+
+label:hover {
+  border: 1px solid #e1e7ed;
```
Comment (OleksiiBlezniuk) [inline src/style.css:63]: You have already set this rule to labels, there is no need to set it for hover state again

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+.big-search input {
+  font-size: 16px;
+  padding: 0 0 0 3px ;
+  line-height: 22px;
+}
+
+.small-search input {
+  font-size: 14px;
+  padding: 0 0 0 1px;
```
Comment (OleksiiBlezniuk) [inline src/style.css:55]: Do you really need this small paddings? And if you really need the, why don't just set padding-left?

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+input::placeholder {
+  color: #3e4e61;
+}
+
+label:hover {
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
```
Comment (OleksiiBlezniuk) [inline src/style.css:64]: same here

File: src/style.css [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+label:focus-within {
+  font-family: "Avenir-Heavy" , sans-serif;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  box-sizing: border-box;
```
Comment (OleksiiBlezniuk) [inline src/style.css:73]: usually we set this rule to * selector, also why do you change your box model on focus? set it to default state of element

File: src/style.css [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+label:hover {
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+label:focus-within {
+  font-family: "Avenir-Heavy" , sans-serif;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
```
Comment (OleksiiBlezniuk) [inline src/style.css:71]: You have already set this rule to labels, don't repeat yourself

**Review by OleksiiBlezniuk — CHANGES_REQUESTED** [review]
Please check your indentations, also there is a problem with your demo page, have you deployed your solution?

File: .linthtmlrc.json [hunk 12]
```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
+  "attr-bans": [
+    "align",
```
[the remark this answers is not in the document]
Comment (barsuchellos) [reply .linthtmlrc.json:3]: sry, I forgot about deploy . About files .json , I will be more careful and before git add I will verify my files !!!)

File: src/style.css [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+  height: 11px;
+  vertical-align: middle;
+  padding: 14px 9px 15px 12px;
+}
+
+.big-input-search ,
+.small-input-search {
+  font-family: inherit;
+  border: none;
+}
```
Comment (OleksiiBlezniuk) [inline src/style.css:49]: Why you decided to disable border? Also you can apply font-family prop to `search` class, no need to duplicate it for this classes

File: src/style.css [hunk 14]
```diff
@@ -0,0 +1,75 @@
...
+  color: #3e4e61;
+}
+
+.search:hover {
+  border-radius: 4px;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus-within {
+  font-family: "Avenir-Heavy" , sans-serif;
```
Comment (OleksiiBlezniuk) [inline src/style.css:73]: You can use `inherit` here

Comment (OleksiiBlezniuk) [reply src/style.css:73]: And you forgot to disable outline

**Review by OleksiiBlezniuk — CHANGES_REQUESTED** [review]
There is a small bug here, fix it please, inputs should have 100% widt

https://user-images.githubusercontent.com/84275496/185065375-56cc0437-2921-4c52-b6e6-a36e7b007dad.mov

h

File: src/style.css (hunk 13 above)
Comment (barsuchellos) [reply src/style.css:49]: in task , when we look on our search , we don't see the input-border , so I decided to disable him . Mb I should to make a white border like `border: 1px solid white;` ?

Comment (OleksiiBlezniuk) [reply src/style.css]: I meant only border
There is a same border for both inputs for all states in figma, check it please. That's why there is no need to disable it

File: src/style.css [hunk 15]
```diff
@@ -0,0 +1,76 @@
...
+.search:hover {
+  border-radius: 4px;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus-within {
+  font-family: inherit;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: none;
+}
```
Comment (OleksiiBlezniuk) [inline src/style.css:76]:
```suggestion
.search:focus-within {
  font-family: inherit;
  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
  outline: none;
}
```

Comment (OleksiiBlezniuk) [reply src/style.css:76]: You forgot to disable outline again 🙂

File: src/style.css (hunk 9 above)
Comment (OleksiiBlezniuk) [reply src/style.css:68]: it can be removed

File: src/style.css (hunk 15 above)
Comment (OleksiiBlezniuk) [reply src/style.css:76]: But now, when you disable your border, your input is jumping

**Review by OleksiiBlezniuk — APPROVED** [review]
Next time read comments a bit more careful😉

File: src/style.css (hunk 15 above)
Comment (barsuchellos) [reply src/style.css:76]: I thought , that otline it's someth other)))) but not like property . I didn't hear about it before this , my bad

**Pull request closed without merging**